__tests__/src/utils/formatters.js

```diff
@@ -31,7 +31,7 @@ describe('prettyWalletBalance', () => {
   const syncedWallet = {
     ethAddress: '0xtestAddress',
     currency: 'ETH',
-    balance: '0.12345678901',
```

Comment:
Can you please remove that change as well? It's better to test that it rounds correctly

Reply:
Fixed